[HDBTI] Nullable DataTime value results into exception

Nullable DataTime value results into the following exception:

**file.csv**:

```
configKey;dateValue;comment
Key1;;Comment
```

> _**Note:**_ `dateValue` is column of type `SECONDDATE` 

**Exception:**
```java
java.time.format.DateTimeParseException: Text '' could not be parsed: Unable to obtain LocalDateTime from TemporalAccessor: {},ISO of type java.time.format.Parsed
	at java.base/java.time.format.DateTimeFormatter.createError(DateTimeFormatter.java:2017) ~[na:na]
	at java.base/java.time.format.DateTimeFormatter.parse(DateTimeFormatter.java:1952) ~[na:na]
	at java.base/java.time.LocalDateTime.parse(LocalDateTime.java:492) ~[na:na]
	at org.eclipse.dirigible.commons.api.helpers.DateTimeUtils.parseDateTime(DateTimeUtils.java:61) ~[dirigible-commons-api-6.1.20.jar:na]
	at com.sap.xsk.hdbti.dao.XSKCSVRecordDao.setValue(XSKCSVRecordDao.java:260) ~[xsk-modules-engines-hdbti-0.13.0-SNAPSHOT.jar:na]
	at com.sap.xsk.hdbti.dao.XSKCSVRecordDao.setPreparedStatementValue(XSKCSVRecordDao.java:227) ~[xsk-modules-engines-hdbti-0.13.0-SNAPSHOT.jar:na]
	at com.sap.xsk.hdbti.dao.XSKCSVRecordDao.insertCsvWithHeader(XSKCSVRecordDao.java:166) ~[xsk-modules-engines-hdbti-0.13.0-SNAPSHOT.jar:na]
	at com.sap.xsk.hdbti.dao.XSKCSVRecordDao.executeInsertPreparedStatement(XSKCSVRecordDao.java:152) ~[xsk-modules-engines-hdbti-0.13.0-SNAPSHOT.jar:na]
	at com.sap.xsk.hdbti.dao.XSKCSVRecordDao.save(XSKCSVRecordDao.java:66) ~[xsk-modules-engines-hdbti-0.13.0-SNAPSHOT.jar:na]
	at com.sap.xsk.hdbti.service.XSKHDBTICoreService.insertCsvRecords(XSKHDBTICoreService.java:75) ~[xsk-modules-engines-hdbti-0.13.0-SNAPSHOT.jar:na]
	at com.sap.xsk.hdbti.processors.XSKHDBTIProcessor.process(XSKHDBTIProcessor.java:116) ~[xsk-modules-engines-hdbti-0.13.0-SNAPSHOT.jar:na]
	at com.sap.xsk.hdbti.synchronizer.XSKTableImportSynchronizer.executeTableImport(XSKTableImportSynchronizer.java:246) ~[xsk-modules-engines-hdbti-0.13.0-SNAPSHOT.jar:na]
	at com.sap.xsk.hdbti.synchronizer.XSKTableImportSynchronizer.reimportAffectedHdbtiFiles(XSKTableImportSynchronizer.java:167) ~[xsk-modules-engines-hdbti-0.13.0-SNAPSHOT.jar:na]
	at com.sap.xsk.hdbti.synchronizer.XSKTableImportSynchronizer.lambda$synchronizeResource$0(XSKTableImportSynchronizer.java:156) ~[xsk-modules-engines-hdbti-0.13.0-SNAPSHOT.jar:na]
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541) ~[na:na]
	at com.sap.xsk.hdbti.synchronizer.XSKTableImportSynchronizer.synchronizeResource(XSKTableImportSynchronizer.java:156) ~[xsk-modules-engines-hdbti-0.13.0-SNAPSHOT.jar:na]
	at org.eclipse.dirigible.core.scheduler.api.AbstractSynchronizer.synchronizeCollection(AbstractSynchronizer.java:107) ~[dirigible-core-scheduler-6.1.20.jar:na]
	at org.eclipse.dirigible.core.scheduler.api.AbstractSynchronizer.synchronizeCollection(AbstractSynchronizer.java:114) ~[dirigible-core-scheduler-6.1.20.jar:na]
	at org.eclipse.dirigible.core.scheduler.api.AbstractSynchronizer.synchronizeCollection(AbstractSynchronizer.java:114) ~[dirigible-core-scheduler-6.1.20.jar:na]
	at org.eclipse.dirigible.core.scheduler.api.AbstractSynchronizer.synchronizeCollection(AbstractSynchronizer.java:114) ~[dirigible-core-scheduler-6.1.20.jar:na]
	at org.eclipse.dirigible.core.scheduler.api.AbstractSynchronizer.synchronizeCollection(AbstractSynchronizer.java:114) ~[dirigible-core-scheduler-6.1.20.jar:na]
	at org.eclipse.dirigible.core.scheduler.api.AbstractSynchronizer.synchronizeCollection(AbstractSynchronizer.java:114) ~[dirigible-core-scheduler-6.1.20.jar:na]
	at org.eclipse.dirigible.core.scheduler.api.AbstractSynchronizer.synchronizeCollection(AbstractSynchronizer.java:114) ~[dirigible-core-scheduler-6.1.20.jar:na]
	at org.eclipse.dirigible.core.scheduler.api.AbstractSynchronizer.synchronizeCollection(AbstractSynchronizer.java:114) ~[dirigible-core-scheduler-6.1.20.jar:na]
	at org.eclipse.dirigible.core.scheduler.api.AbstractSynchronizer.synchronizeRegistry(AbstractSynchronizer.java:91) ~[dirigible-core-scheduler-6.1.20.jar:na]
	at com.sap.xsk.hdbti.synchronizer.XSKTableImportSynchronizer.synchronize(XSKTableImportSynchronizer.java:93) ~[xsk-modules-engines-hdbti-0.13.0-SNAPSHOT.jar:na]
	at org.eclipse.dirigible.core.scheduler.api.AbstractSynchronizerJob$1.run(AbstractSynchronizerJob.java:52) ~[dirigible-core-scheduler-6.1.20.jar:na]
	at org.eclipse.dirigible.commons.timeout.TimeLimited$1.call(TimeLimited.java:38) ~[dirigible-commons-timeout-6.1.20.jar:na]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
	at java.base/java.lang.Thread.run(Thread.java:829) ~[na:na]
Caused by: java.time.DateTimeException: Unable to obtain LocalDateTime from TemporalAccessor: {},ISO of type java.time.format.Parsed
	at java.base/java.time.LocalDateTime.from(LocalDateTime.java:461) ~[na:na]
	at java.base/java.time.format.Parsed.query(Parsed.java:235) ~[na:na]
	at java.base/java.time.format.DateTimeFormatter.parse(DateTimeFormatter.java:1948) ~[na:na]
	... 30 common frames omitted
Caused by: java.time.DateTimeException: Unable to obtain LocalDate from TemporalAccessor: {},ISO of type java.time.format.Parsed
	at java.base/java.time.LocalDate.from(LocalDate.java:396) ~[na:na]
	at java.base/java.time.LocalDateTime.from(LocalDateTime.java:456) ~[na:na]
	... 32 common frames omitted
```

--- a/modules/engines/engine-hdbti/src/main/java/com/sap/xsk/hdbti/dao/XSKCSVRecordDao.java
+++ b/modules/engines/engine-hdbti/src/main/java/com/sap/xsk/hdbti/dao/XSKCSVRecordDao.java
@@ -257,7 +257,7 @@ public class XSKCSVRecordDao implements IXSKCSVRecordDao {
             throws SQLException {
         logger.trace("setValue -> i: " + i + ", dataType: " + dataType + ", value: " + value);
 
-        if (value == null) {
+        if (value == null || value.isEmpty()) {
             preparedStatement.setNull(i, dataType);
         } else if (Types.VARCHAR == dataType) {
             preparedStatement.setString(i, sanitize(value));

--- a/modules/engines/engine-hdbti/src/main/java/com/sap/xsk/hdbti/dao/XSKCSVRecordDao.java
+++ b/modules/engines/engine-hdbti/src/main/java/com/sap/xsk/hdbti/dao/XSKCSVRecordDao.java
@@ -257,7 +257,7 @@ public class XSKCSVRecordDao implements IXSKCSVRecordDao {
             throws SQLException {
         logger.trace("setValue -> i: " + i + ", dataType: " + dataType + ", value: " + value);
 
-        if (value == null || value.isEmpty()) {
+        if (value == null || (value.isEmpty() && (dataType == Types.DATE || dataType == Types.TIME || dataType == Types.TIMESTAMP))) {
             preparedStatement.setNull(i, dataType);
         } else if (Types.VARCHAR == dataType) {
             preparedStatement.setString(i, sanitize(value));

